.github/workflows/native-image.yaml

```diff
@@ -69,12 +73,12 @@
           key: ${{ runner.os }}-graalvm-${{env.GRAALVM_VERSION}}
           restore-keys: |
             ${{ runner.os }}-graalvm-
-      - uses: ilammy/msvc-dev-cmd@v1.4.1
-      - uses: DeLaGuardo/setup-graalvm@8bbfe44ef9c6f5c07e5af036a1bffd561c037d18
+      - uses: ilammy/msvc-dev-cmd@v1.10.0
+      - uses: DeLaGuardo/setup-graalvm@5.0
         with:
-          graalvm-version: '${{env.GRAALVM_VERSION}}'
-      - run: Invoke-Expression -Command "$Env:JAVA_HOME/bin/gu install native-image"
-      - run: .\mvnw.cmd package -Dnative -DskipTests -D cbi.jarsigner.skip=true
+          graalvm: ${{env.GRAALVM_VERSION}}
+          java: ${{env.GRAALVM_JAVA}}
+      - run: .\mvnw.cmd -B package -Dnative -DskipTests -D cbi.jarsigner.skip=true
```

Comment:
Also, I noticed a typo here that is addressed in 7cab4accbc778119518e1ead3fc05b5cdbc3238e . May want to rebase, or just adjust so the rebase works without issue.